### DISCUSSION TITLE
Old "Issues"

### DISCUSSION THREAD

**Discussion by ajitup73** [epochpic/epoch#187] [General]
if one wants to refer old "Issues" and discussions, where can one find them?

**Comment by TomGoffrey** [comment]
The old GitLab server will remain unavailable indefinitely. Unfortunately this is beyond our control.

@keithbennett put quite a bit of effort into porting recent (within the last year IIRC) issues to GitHub - the majority of the 100 or so open issues on here are ported from the old server, so you can search for a previous issue there.

We didn't move across older issues because they typically referred to old versions of EPOCH, so are no longer relevant. However, if you know of a historical issue that you think is pertinent to a current one, and you can't find in on GitHub, tag me, and I'll do a search on the old server for you.
